client/src/store/jobStore.js

```diff
@@ -14,7 +14,7 @@ const getters = {
 
 const actions = {
     fetchJob: async ({ commit }, jobId) => {
-        const { data } = await axios.get(`${getAppRoot()}api/jobs/${jobId}?full=true`);
+        const { data } = await axios.get(`${getAppRoot()}api/jobs/${jobId}?full=true&view=workflow`);
```

Comment:
This is going to fetch the expensive details for every job here. You need to make the view a variable.

Reply:
addressed